### PR TITLE
convert workflowCodeRepository field into a clickable link 

### LINKED ISSUE #10339: Computational Workflow metadata block bug: links don't link
In the Computational Workflow metadata block, the field for "External Code Repository URL" requires a valid URL, but then does not present a link in the UI.  

**What steps does it take to reproduce the issue?**

* When does this issue occur?
When adding a URL to the "External Code Repository URL" field in the Computational Workflow metadata block. 

* Which page(s) does it occurs on?
The dataset page, under the Metadata tab. 

* What happens?
Plain text is displayed.

* To whom does it occur (all users, curators, superusers)?
All users. 

* What did you expect to happen?
A clickable link to be displayed that takes the user to the URL location. 

**Which version of Dataverse are you using?**
6.1

**Any related open or closed issues to this bug report?**
No

**Screenshots:**
Screenshot 2024-02-22 at 4 45 40 PM

--

Screenshot 2024-02-22 at 4 46 29 PM

### PR DESCRIPTION
**Pull request by kmika11**
Add href tag to display url in computational workflow metadata block

**What this PR does / why we need it**: Adds `href` tag to the computational workflow tsv file to properly display and link the url required in the External Code Repository URL field.

**Which issue(s) this PR closes**: 

- Closes #10339

**Special notes for your reviewer**: Redid [https://github.com/IQSS/dataverse/pull/10439](https://github.com/IQSS/dataverse/pull/10439) on new branch and fixed white space changes.

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**: Makes link clickable on dataset page.

**Is there a release notes update needed for this change?**:

**Additional documentation**:

### CODE REVIEW

**Review by pdurbin — COMMENTED** [review]
A little feedback.

File: doc/release-notes/10339-workflow.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The computational workflow metadata block has been updated to display the link for the External Code Repository URL field.
```
[the remark this answers is not in the document]
Comment (kmika11) [reply doc/release-notes/10339-workflow.md:1]: That sounds good to me

[the remark this answers is not in the document]
Comment (pdurbin) [reply doc/release-notes/10339-workflow.md:1]: @kmika11 cool. If you want, you can click "commit suggestion".

**Comment by jggautier** [comment]
@kmika11 How'd you resolve those white space issues?

**Comment by kmika11** [comment]
I "copied raw file" from github [https://github.com/IQSS/dataverse/blob/develop/scripts/api/data/metadatablocks/computational_workflow.tsv](https://github.com/IQSS/dataverse/blob/develop/scripts/api/data/metadatablocks/computational_workflow.tsv) and pasted into a text editor and made the edit there before saving to a .tsv. But, I bet the google sheet strategy would work too. As long as you don't open it in a spreadsheet software (like excel) to make the edits

**Review by scolapasta — APPROVED** [review]

**Comment by qqmyers** [comment]
As is, it looks like the PR both makes the URL into a link and adds quotes around it: 
image
Is that as intended or should the quotes be removed?:
image

**Comment by pdurbin** [comment]
I don't think we want the quotes. Good catch, @landreev

File: scripts/api/data/metadatablocks/computational_workflow.tsv [hunk 2]
```diff
@@ -2,7 +2,7 @@
 	computationalworkflow		Computational Workflow Metadata
 #datasetField	name	title	description	watermark	 fieldType	displayOrder	displayFormat	advancedSearchField	allowControlledVocabulary	allowmultiples	facetable	displayoncreate	required	parent	metadatablock_id	termURI
 	workflowType	Computational Workflow Type	The kind of Computational Workflow, which is designed to compose and execute a series of computational or data manipulation steps in a scientific application		text	0		TRUE	TRUE	TRUE	TRUE	TRUE	FALSE		computationalworkflow
-	workflowCodeRepository	External Code Repository URL	A link to the repository where the un-compiled, human readable code and related code is located (e.g. GitHub, GitLab, SVN)	https://...	url	1		FALSE	FALSE	TRUE	FALSE	TRUE	FALSE		computationalworkflow
+	workflowCodeRepository	External Code Repository URL	A link to the repository where the un-compiled, human readable code and related code is located (e.g. GitHub, GitLab, SVN)	https://...	url	1	<a href=""#VALUE"" target=""_blank"">#VALUE</a>	FALSE	FALSE	TRUE	FALSE	TRUE	FALSE		computationalworkflow
```
Comment (pdurbin) [inline scripts/api/data/metadatablocks/computational_workflow.tsv:5]:
```suggestion
	workflowCodeRepository	External Code Repository URL	A link to the repository where the un-compiled, human readable code and related code is located (e.g. GitHub, GitLab, SVN)	https://...	url	1	<a href="#VALUE" target="_blank">#VALUE</a>	FALSE	FALSE	TRUE	FALSE	TRUE	FALSE		computationalworkflow
```

Untested. Just asking: do we need "" or is " enough?

**Pull request merged**